src/display/annotation_layer.js

```diff
@@ -33,6 +33,8 @@ import {
 import { AnnotationStorage } from "./annotation_storage.js";
 import { ColorConverters } from "../shared/scripting_utils.js";
 
+const DEFAULT_TAB_INDEX = 100;
```

Comment:
Please pick a slightly larger default value, e.g. at least 200, since this is *very* close to the current highest value in https://github.com/mozilla/pdf.js/blob/master/web/viewer.html and any (future) buttons added to the UI *could* easily result in a clash.